stop-es-cluster.sh

```diff
@@ -0,0 +1,12 @@
+kubectl config use-context elk
+kubectl delete deployment es-data --cascade
+kubectl delete deployment es-client --cascade
+kubectl delete deployment es-master --cascade
+kubectl delete deployment kibana --cascade
+kubectl delete rc --all --cascade
+kubectl delete pod --all --cascade
+kubectl delete svc --all
```

Comment:
All these delete operations are only safe if you're running on a context that has only this stuff running.